TM1py/Services/CellService.py

```diff
@@ -188,9 +188,8 @@ def get_value(self, cube_name: str, element_string: str, dimensions: List[str] =
         """
         mdx_template = "SELECT {} ON ROWS, {} ON COLUMNS FROM [{}]"
         mdx_rows_list = []
-        from TM1py.Services.CubeService import CubeService
         if not dimensions:
-            dimensions = CubeService(self._rest).get(cube_name).dimensions
+            dimensions = self.get_dimension_names_for_writing(cube_name=cube_name)
```

Comment:
Good Catch. I think the previous code would have failed in a model with sandboxes as a first dimension.